php/libraries/NDB_Caller.class.inc

```diff
@@ -148,7 +149,13 @@ class NDB_Caller extends PEAR
 
                 $this->type = 'form';
                 return $html;
-            }
+            } /*else{ //if($this->existsAndRequire("NDB_Menu_Filter_Form_$test_name.class.inc")){
```

Comment:
Can you remove this? We shouldn't have commented out code in the codebase, and shouldn't have HTML in the PHP..